### DISCUSSION TITLE
bug nightlight

### DISCUSSION THREAD

**Discussion by AizeenAK** [ChrisTitusTech/winutil#2860] [Q&A]
Hello guys, so when I run the standard, sometimes my Windows Start Menu breaks and crashes, but I know how to fix it. However, my night light doesn’t work, and I don’t know how to fix it. I'm currently using Windows 10 LTSC. Are you guys having this problem as well? Can you help me fix the night light issue?

**Comment by RyMaxz** [comment]
Have you found a solution to this problem?
